install/resources/mybb_theme.xml

```diff
@@ -3966,14 +3966,14 @@ var announcement_quickdelete_confirm = "{$lang->announcement_quickdelete_confirm
 {$forums}
 </table>
 <br />]]></template>
-		<template name="forumdisplay_thread" version="1813"><![CDATA[<tr class="inline_row">
+		<template name="forumdisplay_thread" version="1823"><![CDATA[<tr class="inline_row">
 	<td align="center" class="{$bgcolor}{$thread_type_class}" width="2%"><span class="thread_status {$folder}" title="{$folder_label}">&nbsp;</span></td>
 	<td align="center" class="{$bgcolor}{$thread_type_class}" width="2%">{$icon}</td>
 	<td class="{$bgcolor}{$thread_type_class}">
 		{$attachment_count}
 		<div>
           <span>{$prefix} {$gotounread}{$thread['threadprefix']}<span class="{$inline_edit_class} {$new_class}" id="tid_{$inline_edit_tid}"><a href="{$thread['threadlink']}">{$thread['subject']}</a></span>{$thread['multipage']}</span>
-			<div class="author smalltext">{$thread['profilelink']}</div>
+			<div class="author smalltext">{$thread['profilelink']} &raquo; {$thread['start_datetime']}</div>
```

Comment:
I don't understand the use of `&raquo;` as the separator? I think `|` should be used to be consistent with the rest of the templates (eg usercp_forumsubscriptions_forum) or have the date in parentheses.

Reply:
> I don't understand the use of `&raquo;` as the separator?

Simply because (1) @Eldenroot used it in the screenshot at the top of his original PR, and (2) the View Unread Posts plugin uses it in similar contexts (e.g., for search result listings), so I figured it was something of a standard or at least an approach with a couple of precedents.

That `|` occurs in a different context: options for the user to click, rather than variables being displayed, so I'm not sure that that's a precedent. It also has the association of "this _or_ that", which seems inapplicable in this context.

Parentheses are an interesting idea but also seem to have an inapplicable association: that the parenthesised content is somehow incidental or less relevant than that which precedes it.

If we're to do away with the `&raquo;`, then maybe we don't need to replace it with anything at all? The username preceding it is already styled as a link, so maybe the datetime is already adequately distinguished from it?

All of that said, I'm not particularly invested in any particular option and am happy to go with whatever you and others think is best.

Reply:
I hadn't noticed it in the other PR before because it wasn't in the code.

A particular plugin using it isn't a strong argument for the core imo but I do agree that `|` is used for a set of links as this or that. My main concern with `&raquo;` is it seems to be used mostly for page links (eg in calendar and pagination `{$lang->next} &raquo;` etc).

I do like the idea of no separator. After further looking through the templates I found `reputation_vote` uses a `-` to separate between reputation and last updated date.

Just made these screenshots for convenience and I think `-` seems the best

![image](https://user-images.githubusercontent.com/2552068/80964099-2f241d80-8e08-11ea-92e7-9b6213ee6d86.png)
![image](https://user-images.githubusercontent.com/2552068/80965080-e0778300-8e09-11ea-86a8-a8ae72cf8f52.png)
![image](https://user-images.githubusercontent.com/2552068/80965540-9773fe80-8e0a-11ea-8451-f9a698d3020c.png)
![image](https://user-images.githubusercontent.com/2552068/80964526-ef116a80-8e08-11ea-8ef9-0d711f2f179f.png)

From seeing how it looks, even with the date in the .smalltext element it feels too over emphasised. Perhaps a colour like `#666` from `.post .post_head span.post_date` would work well here too.

I'm just keen to get the style of this right since the froumdisplay thread listing is quite a key part of the experience of the forum users.

Reply:
> My main concern with `&raquo;` is it seems to be used mostly for page links (eg in calendar and pagination `{$lang->next} &raquo;` etc).

Yep, I agree that that's a concern.

> Just made these screenshots for convenience and I think `-` seems the best

While I prefer `-` to `|` and parens, I think I prefer most of all just adding an &nbsp; immediately after the username to add some extra spacing, and otherwise not using any separator at all. Can you try that and see what you think?

> Perhaps a colour like `#666`

Seems to look fine although I'd go with `#555` instead mostly because... connotations...

> I'm just keen to get the style of this right since the froumdisplay thread listing is quite a key part of the experience of the forum users.

Yep, fair enough.

Reply:
A thoughtless suggestion: `{datetime} {$lang->by} {author}`..

Reply:
> Seems to look fine although I'd go with `#555` instead mostly because... connotations...

Aha I only suggested it as it's used for the dates on posts. But I agree it is a bit light so `#555` is probably better.

> adding an `&nbsp;` immediately after the username to add some extra spacing

That's what the first screenshot shows and fees too cluttered to me.

> A thoughtless suggestion: `{datetime} {$lang->by} {author}`..

I like the idea but I worry that it makes it harder to scan down the list of threads than it is at the moment with thread topic and author aligned to the left. Again it is too much emphasis on the date...
![image](https://user-images.githubusercontent.com/2552068/80994583-c56e3880-8e34-11ea-966a-34cce184977a.png)

Reply:
> > adding an `&nbsp;` immediately after the username to add some extra spacing
> 
> That's what the first screenshot shows and fees too cluttered to me.

Mmm, it seems to me that your first screenshot only shows a _single_ space, whereas I'm suggesting an `&nbsp;` _followed_ by a single space. That adds an extra little bit of differentiation (in the form of spacing) which, for me, makes the difference, and makes that the best option. The fewer bits of information a reader has to process - commas, dashes, parentheses, etc - the easier it is for her/him to skim the listing.



> > A thoughtless suggestion: `{datetime} {$lang->by} {author}`..
> 
> I like the idea but I worry that it makes it harder to scan down the list of threads than it is at the moment with thread topic and author aligned to the left. Again it is too much emphasis on the date...

Agreed. And for a "thoughtless" suggestion, it's very thoughtful (one boggles at the power of thought if this is not it!), but it doesn't quite hit the mark. I sense and believe that the author should be leftmost in the listing, rather than the date.

Reply:
Paging @justinsoltesz is my answer for any questions layout/design related 😉

Reply:
Good one, @euantoro. We do need a qualified circuit-breaker!

Reply:
Thinking about this in the context of 1.9's theme, I would suggest a comma:

<img width="409" alt="Screen Shot 2020-05-05 at 10 05 33 AM" src="https://user-images.githubusercontent.com/544942/81075503-74a02380-8eb8-11ea-88cf-70b2e0f60e8a.png">
Note that the lighter text color there is `#999` with the link color `#666`

This is 1.8 with a comma and `#999`:
<img width="256" alt="Screen Shot 2020-05-05 at 10 07 06 AM" src="https://user-images.githubusercontent.com/544942/81075565-8681c680-8eb8-11ea-9658-cb77469a5ce9.png">

Reply:
Thanks, @justinsoltesz. Perhaps, then, we go with a comma and font colour of #999, with that colour set through a class, and selected in global.css? At the very least, this solution eases the transition to 1.9.

Reply:
Yeah, now the only problem is that CSS updates aren't implemented in the upgrade script in 1.8 😢 I wonder if there's some existing CSS class we can use that gets close enough..?

Reply:
I can't see one...

Is a temporary inline style (on top of the CSS that won't get updated) too naughty?

Reply:
I think what we should do is add a style to `global.css` and make a note in the release notes for users. That's probably the best way all round.

Reply:
It's not a big issue if an administrator doesn't add the style when upgrading so I agree that's best.

Reply:
Agreed, it's not a breaking change if the administrator doesn't add it.

Reply:
OK, see what you guys think of my latest commit.